templates/documents/general_list.html

```diff
@@ -10,7 +10,7 @@
 <html>
 <head>
 
-{headerTemplate assets='datetime-picker|jquery-ui|jquery-ui-lightness'}
```

Comment:
Seeing following js console error at Miscellaneous->New Documents:

![image](https://user-images.githubusercontent.com/278968/75643078-a237ca80-5bf2-11ea-9f98-dca22eb77b2f.png)

Reply:
autocomplete is probably in another file stemming off of this file. Also there is still internal stylesheets in this file that need to be removed as they were for jQuery UI.